codesearch/performance/performance.go

```diff
@@ -107,11 +107,10 @@ func (t *Tracker) Add(key label, value int64) {
 
 func (t *Tracker) Keys() []label {
 	t.mu.Lock()
-	keys := maps.Keys(t.data)
+	keysIter := maps.Keys(t.data)
```

Comment:
Is it safe to leak the iterator outside the locked region?

Reply:
it's not safe. I updated the code here and also in copy_on_write.go